ddtrace/contrib/pytest/plugin.py

```diff
@@ -164,23 +192,27 @@ def _start_test_module_span(item):
     test_module_span.set_tag_str(_EVENT_TYPE, _MODULE_TYPE)
     test_module_span.set_tag_str(_SESSION_ID, str(test_session_span.span_id))
     test_module_span.set_tag_str(_MODULE_ID, str(test_module_span.span_id))
-    test_module_span.set_tag_str(test.MODULE, _get_module_name(item))
+    test_module_span.set_tag_str(test.MODULE, _get_module_name(item, is_package))
     test_module_span.set_tag_str(test.MODULE_PATH, _get_module_path(item))
-    _store_span(item, test_module_span)
-    return test_module_span
+    if is_package:
+        _store_span(item, test_module_span)
+    else:
+        _store_module_span(item, test_module_span)
+    return test_module_span, is_package
 
 
-def _start_test_suite_span(item):
+def _start_test_suite_span(item, test_module_span=None):
```

Comment:
I'd lean towards not letting `test_module_span` be optional.

Whatever creates a `test_suite_span` should properly initialize the module (and session) spans prior to creating the suite span, and there should be no guessing.